docs/directives.md

```diff
@@ -882,7 +882,7 @@ type Mutation {
 
 ### query
 
-Represents the API call's query parameters, either as a static object or with dynamic parameters using Mustache templates. These parameters append to the URL.
+Represents the API call's query parameters, either as a static object or with dynamic parameters using Mustache templates. These parameters append to the URL. The `key` parameter of `query` is optional. If it is not present TailCall uses the last item from `batchKey` parameter of `@http` directive to derive the `key` parameter, so remember to adjust the order of the items accordingly.
```

Comment:
I think this note should be written under an `:::important` section of `batchKey`

Reply:
Let's provide an actual example here for more clarity.